docs/chitter.rb

```diff
@@ -0,0 +1,26 @@
+require 'pg'
+require_relative 'peep'
+
+class Chitter
+  attr_reader :peep_log
+
+  def initialize
+    @peep_log = []
+  end
+
+  def self.all
```

Comment:
It looks like you were in the process of creating a database to store peeps, rather than as an array in Chitter. In your readme, you say that the challenge is only partially complete - I think it would benefit from mentioning the specific user stories that are met, to what extent, and what the next steps would have been and why